add a Contributor License Agreement workflow

paulfitz opened the pull request:
Follow up to https://github.com/gristlabs/grist-core/pull/1546

This adds a workflow to point new contributors to the CLA and check they have accepted it prior to landing PRs. Since the individual CLA agreement refers to executing a Corporate CLA if needed, I added a note on how to execute one.

You can try it out by making a dummy PR to https://github.com/paulfitz/grist-core/

paulfitz committed:
add a Contributer License Agreement workflow

This adds a workflow to point new contributors to the CLA and
check they have accepted it prior to landing PRs. Since the
individual CLA agreement refers to executing a Corporate CLA
if needed, I added a note on how to execute one.

dsagal reviewed (approved):
Tried it out, seems to work fine!

paulfitz committed:
omit contributions from French gov from scope of individual CLA

It is convenient for contributors to be able to sign the individual
CLA without including French government contributions.
